src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,40 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
```

Comment:
Regarding the second "if" added for not breaking an existing functional test, I'd suggest to add a comment above the "if" with the test(s) names that such "if" is preventing to fail.